src/solver/optimisation/adequacy_patch_csr/solve_problem.h

```diff
@@ -0,0 +1,15 @@
+
+#pragma once
```

Comment:
Al this was previously in **opt_fonctions.h**, which causes re-compilation of many files apparently unrelated to changes we made.